addons/core/functions/fnc_getLrSettings.sqf

```diff
@@ -86,6 +86,18 @@ if (isNil "_radioCode") then {
     _value set [TFAR_CODE_OFFSET, _radioCode];
 };
 
+private _halfDuplexOverride = _value select HALFDUPLEX_OVERRIDE_OFFSET;
+if (isNil "_halfDuplexOverride") then {
+    _halfDuplexOverride = false; //Override is disabled by default. No harm done if we don't find the config value since it's expected default behaviour anyways
+    private _override = [_radio_object, "tf_halfDuplexOverride"] call TFAR_fnc_getLrRadioProperty;
```

Comment:
Not that I am aware of, but I might also be reading something wrong... [The current version](https://github.com/michail-nikolaev/task-force-arma-3-radio/blob/1.0/addons/core/functions/fnc_getLrRadioProperty.sqf#L22) only lists two params though, `_radio` and `_property`.